test/json.go

```diff
@@ -0,0 +1,24 @@
+package test
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+)
+
+// ReadJSON is a testing utility that reads data from JSON file into an
+// interface and fails with FailNow on error.
+//
+//   cfg := Config{}
+//   test.ReadJSON(t, "testdata/config.json", &cfg)
+func ReadJSON(t testing.TB, fname string, v interface{}) {
+	t.Helper()
+	f, err := os.Open(fname) // nolint:gosec
+	if err != nil {
+		t.Fatalf("cannot open %s: %v", fname, err)
+	}
+	defer f.Close() //nolint
```

Comment:
best to name the linters I think, so `//nolint:errcheck,gosec` (not sure about gosec, sometimes I get that one, sometimes I dont)